pyqtgraph/parametertree/parameterTypes/actiongroup.py

```diff
@@ -58,3 +60,19 @@ def setButtonOpts(self, **opts):
         buttonOpts = self.opts.get("button", {}).copy()
         buttonOpts.update(opts)
         self.setOpts(button=buttonOpts)
+
+
+class ActionGroup(ActionGroupParameter):
+    sigActivated = QtCore.Signal()
+
+    def __init__(self, **opts):
+        warnings.warn(
+            "`ActionGroup` is deprecated and will be removed in the first release after "
+            "January 2023. Use `ActionGroupParameter` instead. See "
+            "https://github.com/pyqtgraph/pyqtgraph/pull/2505 for details.",
+            DeprecationWarning,
```

Comment:
probably want to add `stacklevel=2` after `DeprecationWarning`; see any of our usage of deprecation warnings in the library 😆 (it's an awful default in the standard lib)